ui/components/ui/list-item/index.scss

```diff
@@ -7,6 +7,7 @@
 
   @include Paragraph;
 
+  border-top: 1px solid var(--color-border-muted);
```

Comment:
We removed the showBorder condition here and that resulted in the border top in activity list. I think we can remove it permanently now from css. Since we are removing the feature flag?
![Screenshot 2023-06-01 at 9 04 23 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/3d65b824-ffa6-4f95-bdc7-cb27a01eb8d3)


```suggestion

```